fvm/evm/emulator/emulator.go

```diff
@@ -305,7 +306,10 @@ func (bl *BlockView) DryRunTransaction(
 	// use the from as the signer
 	msg.From = from
 	// we need to skip nonce check for dry run
-	msg.SkipAccountChecks = true
+	// Previous we had: msg.SkipAccountChecks = true,
+	// so we need to set SkipNonceChecks, SkipFromEOACheck both to true.
```

Comment:
This is great to know, but probably not something we need to keep in the code. Maybe just mentioning it in the PR description would be enough.

Reply:
Good point. Removed in https://github.com/onflow/flow-go/pull/7241/commits/de0a491b2000aea84f210d34a4aa27e0e196a66b .